### DISCUSSION TITLE
Speaker Diarization

### DISCUSSION THREAD

**Discussion by k3ybladewielder** [speechbrain/speechbrain#1762] [Q&A] [answered]
What is the pre trained model for the speaker diarization task ?
the speaker diarization task is in the performance benchmarking but dont is on the pretrained models list

**Comment by louismorgner** [comment]
I was also looking for that. As an alternative, I have started to play around with speaker recognition with which you can in theory build your own diarization workflow.

**Comment by mravanelli** [comment, marked as answer]
Hi,
the team is working on a diarization interface that will combine pretrained
VAD, pretrained speaker embeddings, and clustering. We are currently
testing and improving it.
I hope it will be available quite soon.

**Reply by PhantomSpike** [reply]
Hi,

I had exactly the same Q.

Can you please let us know here in the comments or otherwise when these pre-trained models become available?

Thanks!

**Reply by Aaryan369** [reply]
Hi @mravanelli ,
Thanks for creating such an awesome package! Is there any approximate date by which we can expect the diarization feature?
